pkg/util/codec/BUILD.bazel

```diff
@@ -12,9 +12,9 @@ go_library(
     importpath = "github.com/pingcap/tidb/pkg/util/codec",
     visibility = ["//visibility:public"],
     deps = [
+        "//pkg/errctx",
         "//pkg/parser/mysql",
         "//pkg/parser/terror",
-        "//pkg/sessionctx/stmtctx",
```

Comment:
Once this PR is merged. I can continue this process on `rowcodec`, `tablecodec`... They generally also just need `typeCtx` and `errCtx`.